app/views/works/new_import.html.erb

```diff
@@ -91,16 +98,28 @@
             <%= check_box_tag "restricted" %>
             <%= label_tag "restricted", ts("Lock all imports to registered users only.") %>
           </li>
-          <li>
-            <%= check_box_tag "override_tags" %>
-            <%= label_tag "override_tags", ts("Set the following tags on all works, overriding whatever the importer finds.") %>
-          </li>
         </ul>
       </dd>
+      <dt><%= ts("Override tags and notes") %></dt>
+      <dd>
+        <%= check_box_tag "override_tags" %>
+        <%= label_tag "override_tags", ts("Set the following tags and/or notes on all works, overriding whatever the importer finds in the content.") %>
+      </dd>
     </dl>
   </fieldset>
 
-  <%= render :partial => 'work_form_tags', :locals => {:include_blank => true} %>
+  <%= render :partial => 'work_form_tags', locals: { include_blank: true, import: true } %>
+
+  <fieldset>
```

Comment:
If you have a second, could you fix the indenting here? `<legend>` has extra indenting and lines 118 and 119 should both be indented two more spaces. (Line 67 above could probably use a `ts()` as well, but I'm not particularly concerned given the future of `ts()`.)

Going to mark ready to merge, though, since it's probably not worth holding it up for either of those things.